homeassistant/components/stream/core.py

```diff
@@ -294,6 +302,8 @@ async def get(self, request, token, sequence=None):
 
         return await self.handle(request, stream, sequence)
 
-    async def handle(self, request, stream, sequence):
+    async def handle(
+        self, request: web.Request, stream: Stream, sequence: str
+    ) -> web.Response | web.StreamResponse:
```

Comment:
`web.Response` inherits `web.StreamResponse`.
```suggestion
    ) -> web.StreamResponse:
```